src/passkeyProvider.ts

```diff
@@ -36,6 +43,7 @@ export class PasskeyProvider {
   private static _instance: PasskeyProvider = new PasskeyProvider();
   private keyPair: { privateKey: string; publicKey: string } | undefined =
     undefined;
+  private axiosInstance = axios.create();
```

Comment:
generateEd25519KeyPair --> no need for async
why return same value that was passed in ?